crmprtd/insert.py

```diff
@@ -54,6 +56,13 @@ def max_power_of_two(num):
     return 2 ** floor(mathlog(num, 2))
 
 
+@functools.lru_cache(maxsize=None)
```

Comment:
I wanted to use caching since the same database will be logged repeatedly during calls to insert(). The idea was partially inspired by the cached_function in align.py but differs as the return value is not a mapped object. However, since it is the parameter _sesh_ that cannot be cached, the function would still need to be reevaluated each time. I will correct this to not use the lru_cache.